packages/shared/src/lib/core/network/constants/default-explorer-urls.constant.ts

```diff
@@ -4,10 +4,12 @@ import { SupportedNetworkId } from './supported-network-id.constant'
 export const DEFAULT_EXPLORER_URLS: Readonly<{ [key in NetworkId]?: string }> = {
     [SupportedNetworkId.Iota]: 'https://explorer.iota.org',
     [SupportedNetworkId.Shimmer]: 'https://explorer.shimmer.network',
+    [SupportedNetworkId.IotaTestnet]: 'https://explorer.iota.org/iota-testnet',
```

Comment:
because this contains a path, our url building wont work, because it assumes that this is a an origin.